### PR TITLE
Update licenses

### LINKED ISSUE #46: Add License for the Manifesto itself
Shouldn't the content – the manifesto itself - be licensed by CC-BY-4.0 and not MIT? 

If so, we have add another License like in the [docs-page](https://github.com/SovereignCloudStack/docs-page/pull/4)

### PR DESCRIPTION
**Pull request by maxwolfs**
Signed-off-by: maxwolfs <mail@maxwolfs.com>

Question: Should there also be a sentence within the Readme stating something like `All logos are property of their respective owners`?

closes #46

### CODE REVIEW

**Comment by itrich** [comment]
> Question: Should there also be a sentence within the Readme stating something like `All logos are property of their respective owners`?

Yes, definitly! We should move all restricted logos to a dedicated subdirectory and make this clear within the `README`.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Open Operations Manifesto
 
+## Licenses
+
+The content of this repository is published under multiple licenses.
+The manifesto itself is licensed under [CC-BY-SA 4.0](LICENSE-DOCS).
```
Comment (garloff) [inline README.md:6]: I in general like CC-BY-SA-4.0, a weak copyleft license for non-code (and not-compiled code).
For this specific manifesto, I would be fine to drop the SA (share-alike) clause. In other words: If someone wants to copy content from the manifesto and then use it under a restrictive license (internally?) I would not mind. I would keep the attribution clause (BY) though, so credit goes to where credit is due.
It's probably not very relevant -- the manifesto gets its power by being shared, so anyone wanting to create a derived manifesto is likely to publish it to have any effect ...

**Review by garloff — APPROVED** [review]
/lgtm

**Review by fkr — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (itrich) [reply README.md:6]: Same here. Let's ditch the `Share-Alike` feature for the sake of a possible higher adoption rate.

**Review by itrich — CHANGES_REQUESTED** [review]
As discussed, please switch to the CC-BY 4.0 license. Thank you :smiley:

**Review by itrich — APPROVED** [review]
LGTM

**Pull request merged**